ee/hogai/eval/conftest.py

```diff
@@ -1,209 +1,25 @@
-    with django_db_blocker.unblock():
-        team = Team.objects.order_by("-created_at").first()
-        today = datetime.date.today()
-        # If there's no eval team or it's older than today, we need to create a new one with fresh data
-        should_create_new_team = not team or team.created_at.date() < today
-
-        if should_create_new_team:
-            print(f"Generating fresh demo data for evals...")  # noqa: T201
-
-            matrix = HedgeboxMatrix(
-                seed="b1ef3c66-5f43-488a-98be-6b46d92fbcef",  # this seed generates all events
-                days_past=120,
-                days_future=30,
-                n_clusters=500,
-                group_type_index_offset=0,
-            )
-            matrix_manager = MatrixManager(matrix, print_steps=True)
-            with override_settings(TEST=False):
-                # Simulation saving should occur in non-test mode, so that Kafka isn't mocked. Normally in tests we don't
-                # want to ingest via Kafka, but simulation saving is specifically designed to use that route for speed
-                org, team, user = matrix_manager.ensure_account_and_save(
-                    f"eval-{today.isoformat()}", EVAL_USER_FULL_NAME, "Hedgebox Inc."
-                )
-        else:
-            print(f"Using existing demo data for evals...")  # noqa: T201
-            org = team.organization
-            user = org.memberships.first().user
-
-        yield org, team, user
-
-
-@pytest.fixture(scope="package", autouse=True)
-def core_memory(demo_org_team_user, django_db_blocker) -> Generator[CoreMemory, None, None]:
-    initial_memory = """Hedgebox is a cloud storage service enabling users to store, share, and access files across devices.
-
-    The company operates in the cloud storage and collaboration market for individuals and businesses.
-
-    Their audience includes professionals and organizations seeking file management and collaboration solutions.
-
-    Hedgebox's freemium model provides free accounts with limited storage and paid subscription plans for additional features.
-
-    Core features include file storage, synchronization, sharing, and collaboration tools for seamless file access and sharing.
-
-    It integrates with third-party applications to enhance functionality and streamline workflows.
-
-    Hedgebox sponsors the YouTube channel Marius Tech Tips."""
-
-    with django_db_blocker.unblock():
-        core_memory, _ = CoreMemory.objects.get_or_create(
-            team=demo_org_team_user[1],
-            defaults={
-                "text": initial_memory,
-                "initial_text": initial_memory,
-                "scraping_status": CoreMemory.ScrapingStatus.COMPLETED,
-            },
-        )
-    yield core_memory
-
-
-_nodeid_to_results_url_map: dict[str, str] = {}
-"""Map of test nodeid (file + test name) to Braintrust results URL."""
+def setup_evals(django_db_setup):  # noqa: F811
```

Comment:
Looks like a placeholder?

Naming nit: as a verb, "set up" should have a space
```suggestion
def set_up_evals(django_db_setup):  # noqa: F811
```

Reply:
It is used for Django DB setup. Updated the function name, thanks!